docs/externalGrading.md

```diff
@@ -238,36 +238,58 @@ Running PrairieLearn locally with externally graded question support looks somet
+```sh
+docker run -it --rm -p 3000:3000 -v $PWD\:/course -v $HOME\pl_ag_jobs:/jobs -e HOST_JOBS_DIR=/c/Users/Tim/pl_ag_jobs -v /var/run/docker.sock:/var/run/docker.sock prairielearn/prairielearn 
+```
+
+**Note** the following about `HOST_JOBS_DIR` on PowerShell:
+
+* Use Unix-style paths (i.e., use `/c/Users/Tim/pl_ag_jobs`, **not** `C:\Users\Tim\pl_ag_jobs`)
+* Use the full path rather than `$HOME` (i.e., use `/c/Users/Tim/pl_ag_jobs`, **not** `$HOME/pl_ag_jobs`).
 
 ##### Common issues
 
-Windows users might see problems running grading jobs. The most common issue is the:
+Windows users might see problems running grading jobs:
 
 ```bash
 standard_init_linux.go:207: exec user process caused "no such file or directory"
 ```
 
-The error occurs during grading as a result of an OS new-line incompatibility with the `entrypoint` script in the externally
+This error occurs during grading as a result of an OS new-line incompatibility with the `entrypoint` script in the externally
 graded question. One solution for this is to make a `.gitattributes` files in your PL repository with the line
 `*.sh text eol=lf`. This tells the GitHub client to write the script files in native Linux
 format instead of converting them for Windows (which breaks mapping them back into docker). 
 This mimics the [`.gitattributes` file in the main PrairieLearn repo](https://github.com/PrairieLearn/PrairieLearn/blob/master/.gitattributes).
 
+
+```sh
+error: Error processing external grading job 1
+error: handleGraderErrorUnable to launch Docker container for grading: (HTTP code 500) server error - invalid mode: /grade
+```
+
+This error can occur when `HOST_JOBS_DIR` cannot be accessed.
+
+1. Verify that the `pl_ag_jobs` directory was created successfully.
+2. Verify that you are using Unix-style slashes even though you are using PowerShell (i.e., use `-e HOST_JOBS_DIR=/c/Users/Tim/pl_ag_jobs`, **not** `-e HOST_JOBS_DIR=C:\Users\Tim\pl_ag_jobs`).
+
 #### Running locally (native, not on Docker)
 
-When not running in Docker, things are easier. The Docker socket can be used normally, and we're able to store job files automatically. By default, they are stored in `$HOME/pl_ag_jobs` on Unix-based systems and `$USERPROFILE/pl_ag_jobs` on Windows. However, if you run PrairieLearn with a an environment variable `JOBS_DIR=/abs/path/to/my/custom/directory/`, that directory will be used instead. Note that this environment variable has no effect when running on Docker.
+When not running in Docker, things are easier. The Docker socket can be used normally, and we're able to store job files automatically. By default, they are stored in `$HOME/pl_ag_jobs` on Unix-based systems and `$USERPROFILE/pl_ag_jobs` on Windows. However, if you run PrairieLearn with an environment variable `JOBS_DIR=/abs/path/to/my/custom/directory/`, that directory will be used instead. Note that this environment variable has no effect when running on Docker.
```

Comment:
```suggestion
When not running in Docker, things are easier. The Docker socket can be used normally, and we're able to store job files automatically. By default, they are stored in `$HOME/pl_ag_jobs` on Unix-based systems and `$USERPROFILE/pl_ag_jobs` on Windows. However, if you run PrairieLearn with an environment variable `HOST_JOBS_DIR=/abs/path/to/my/custom/directory/`, that directory will be used instead. Note that this environment variable has no effect when running on Docker.
```
Not sure what this line is trying to say. Is this really `JOBS_DIR` or is it supposed to be `HOST_JOBS_DIR`?

Reply:
It's actually `JOBS_DIR`: https://github.com/PrairieLearn/PrairieLearn/blob/master/lib/externalGraderLocal.js#L236

PL actually has two different job directories: one _inside_ the container and one _outside_.

When we are running inside Docker, the directory we use for the job _inside_ the container is hard-coded to look like `/jobs/job_53`, and `HOST_JOBS_DIR` is used to compute the name of the same directory as seen _outside_ Docker by the host (e.g., `C:\Users\Tim\pl_ag_jobs\job_53`).

When running outside of Docker, we don't distinguish between these two directories. We indicate this by _not_ setting `HOST_JOBS_DIR`. We can either: (1) not set `JOBS_DIR` either, in which case we use `~/.pljobs/job_53`, or (2) set `JOBS_DIR`, in which case this directly specifies the location.